element_deeplabcut/dlc.py

```diff
@@ -424,36 +496,56 @@ class PoseEstimationTask(dj.Manual):
     -> VideoRecording
     -> Model
     ---
-    task_mode='load' : enum('load', 'trigger')   # load results or trigger computation
-    pose_estimation_output_dir='': varchar(255)  # output directory relative to the root data directory
+    task_mode='load' : enum('load', 'trigger')  # load results or trigger computation
+    pose_estimation_output_dir='': varchar(255) # output dir relative to the root dir
+    pose_estimation_params       : longblob     # analyze_videos params, if not default

```

Comment:
if we are to allow for users to specify this, we must allow this to be nullable
```suggestion
    pose_estimation_params=null   : longblob     # analyze_videos params, if not default

```

Reply:
Sure, adding this to my most recent commit